### DISCUSSION TITLE
About the OnThrow script in the sample directory

### DISCUSSION THREAD

**Discussion by drDLH** [btraceio/btrace#622] [Q&A]
Hi, everyone. I currently have one problem using the official sample OnThrow.java.

When I use 1.3.11.3 version of Btrace, it work perfectly. It can print out the missing exception if someone forgot to use exception.printStackTrace() in the code.

When I update btrace version to 2.2.2 or 2.2.3, the OnThrow.java seem to stop working, no output in the console at all.

I'm using btrace under windows 10 system, the jdk and jre version is 1.8.0_351. 
The code:

```
	@RequestMapping("/exception")
	public String exception() throws Exception{
		try {
			System.out.println("start...");
			System.out.println(1/0);
			System.out.println("end...");
		}catch(Exception e) {
			//throw exception without print detail.
			e.printStackTrace();
		}
		return "success";
	}
```

Please let me know if someone understand the reason. Thanks a lot!

**Comment by jbachorik** [comment]
Hi @drDLH !

I suspect there are some remaining issues in the custom stack frame map handling.
I will try to take a look at this, but as you can see from my activity on this project, my 'bread-winning' effort does not leave me much time to give to BTrace. But I will do my best to diagnose this problem.
